### ISSUE TITLE
Create documentation on license creation

### ISSUE THREAD

**Issue by fharper** [konstructio/kubefirst-docs#767]
### What type of suggestions?

Missing Information

### What version of the docs?

2.5+

### What is the issue?

Documenting the pro-tier licensing process.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by konstructcarrie** [comment]
@fharper  Here I'm assuming you mean walking a user through how they "upgrade" for more clusters/higher paid tier once they're the console? 

Correct me if i'm wrong, otherwise this content will be incorporated in to the Kubefirst Pro docs 👍

**Comment by fharper** [comment]
Yes, it was about that. It was also before we duplicate the docs to have another one specific to pro, so feel free to do whatever make sense for you with this issue.
